openedx/core/djangoapps/user_authn/serializers.py

```diff
@@ -0,0 +1,76 @@
+"""
+MFE Context API Serializers
+"""
+
+from rest_framework import serializers
+
+
+class ProvidersSerializer(serializers.Serializer):
+    """
+    Providers Serializers
+    """
+
+    id = serializers.CharField(allow_null=True)
+    name = serializers.CharField(allow_null=True)
+    iconClass = serializers.CharField(source='icon_class', allow_null=True)
+    iconImage = serializers.CharField(source='icon_image', allow_null=True)
+    skipHintedLogin = serializers.BooleanField(source='skip_hinted_login', default=False)
+    loginUrl = serializers.CharField(source='login_url', allow_null=True)
+    registerUrl = serializers.CharField(source='register_url', allow_null=True)
+
+
+class PipelineUserDetailsSerializer(serializers.Serializer):
+    """
+    Pipeline User Details Serializers
+    """
+
+    username = serializers.CharField(allow_null=True)
+    email = serializers.CharField(allow_null=True)
+    name = serializers.CharField(source='fullname', allow_null=True)
+    firstName = serializers.CharField(source='first_name', allow_null=True)
+    lastName = serializers.CharField(source='last_name', allow_null=True)
+
+
+class ContextDataSerializer(serializers.Serializer):
+    """
+    Context Data Serializers
+    """
+
+    currentProvider = serializers.CharField(source='current_provider', allow_null=True)
+    platformName = serializers.CharField(source='platform_name', allow_null=True)
+    providers = serializers.ListField(
```

Comment:
why did we not provide the `source` here?

Reply:
Yes.